karate-core/src/test/java/com/intuit/karate/core/KarateHttpMockHandlerTest.java

```diff
@@ -159,7 +159,7 @@ void testKarateRemove() {
         startMockServer();
         run(
                 urlStep(),
-                "path '/hello/1'",
+                "path '/hello', '1'",
```

Comment:
>is there a very good reason to treat the first slash differently

I only thought for backwards compatibility, to soften the blow as it were, but I'm totally fine with removing the workaround, thought I'd still like to keep the warning message, as most likely someone has done this by mistake.

Reply:
One thing I did notice, is that Tomcat does not handle encoded/escaped `/` very well, but Jetty does.. I tried switching from Tomcat to Jetty but ran into another bug.. 

https://github.com/eclipse/jetty.project/issues/6196

Seems like Spring MVC is doing the wrong thing here

So if we fix this handling and let `URIBuilder` encode `/` to till it's hearts content, we cannot test it.. because tomcat will cause trouble and jetty triggers a bug in spring mvc..

That's why none of the new `encoding.feature` tests include `/` for now..